geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/TypologyClassification.groovy

```diff
@@ -189,7 +189,7 @@ IProcess identifyLczType() {
                                         AS SELECT   $ID_FIELD_RSU,
                                                 CASE WHEN IMPERVIOUS_FRACTION_LCZ>PERVIOUS_FRACTION_LCZ AND IMPERVIOUS_FRACTION_LCZ>0.05
```

Comment:
it is necessary to add some conditions here, other wise the impervious_fraction may be larger than water and than vegetation but it will not be affected with impervious. Here is the replacement piece of query:
CASE WHEN IMPERVIOUS_FRACTION_LCZ>ALL_VEGETATION AND IMPERVIOUS_FRACTION_LCZ>WATER_FRACTION_LCZ AND IMPERVIOUS_FRACTION_LCZ>0.05